### ISSUE TITLE
ENOENT: no such file or directory. v2 deployment

### ISSUE THREAD

**Issue by JohnatanNorman** [zeit/now-examples#274]
Hi there!
I was using **@google-cloud/storage** and I made **json auth file** file statically build so it would appear after build step.
![image](https://user-images.githubusercontent.com/41754671/53120861-d69a9480-3563-11e9-8f64-e859683c8173.png)
As expected it appears in output but I still get this message that file doesn't exist.
![image](https://user-images.githubusercontent.com/41754671/53120820-c08cd400-3563-11e9-8a38-5dd355820450.png)
I found the thread on the forums that you can build this google cloud file from **now secrets** with base64 encryption, but I don't have a problem with github that the guy had, and I don't like that workaround. Is there other option to make this file visible (will it work if I use it from /tmp?) or _I have to_ implement solution with now secrets?
Thanks in advance!

**Comment by timothyis** [comment]
Hi @JohnatanNorman! Is this relating to one of our examples? I'm not sure how your project is laid out. Perhaps you could contact support@zeit.co for our support team to look into your issue if it is not related to an example.

**Comment by degroote22** [comment]
When importing the auth.json file you have to use __dirname and resolve the path from this location to let NCC know that you're using this external file.

I'm doing it like this:
```typescript
const FIREBASE_SERVICE_ACCOUNT = require(__dirname + "/serviceAccountKey.json");
```

**Comment by JohnatanNorman** [comment]
@degroote22 I am not sure I understand where I will put this. I was creating the storage like this
![image](https://user-images.githubusercontent.com/41754671/53194545-a830be00-3624-11e9-8c82-990233839691.png) Is there a better way to do this? Where do I put this file that you required?

@timothyis I am sorry for this, it's not about one of your examples, so I will close the issue. Thanks for the help link!

**Comment by degroote22** [comment]
It's the same idea. That's how I'm doing it.

```typescript
const storage = new Storage({
  projectId: "xxxxx",
  keyFilename: __dirname + "/serviceAccountKey.json",
});
```

**Comment by degroote22** [comment]
@timothyis This issue with NCC and __dirname is a very common one.
I spent a couple of hours to solve it and that's why I'm answering these questions when I see one.
Consider explaining it better and documenting it. I don't even remember and can't find where I saw this tip.

**Comment by timothyis** [comment]
I appreciate the help @degroote22! I'll pass this on to the ncc team.

**Comment by degroote22** [comment]
> I appreciate the help @degroote22! I'll pass this on to the ncc team.

Also recommend people to try and compile/run their code locally with ncc and explain that the builders use ncc because I don't remember where I saw it aswell.
I also spent some hours debugging ncc issues by deploying code to now and running locally with ts-node. When I understood that the builders were using ncc I started to test my code locally with ncc and could fix issues way faster.

Thanks for your awesome work on now, ncc, etc.
I love it ❤️
